src/main/target/OMNIBUSF4/target.h

```diff
@@ -174,6 +174,10 @@
 
 #define AVOID_UART1_FOR_PWM_PPM
 
+#define SPEKTRUM_BIND
+// USART1 RX
+#define BIND_PIN                PA10
```

Comment:
Thanks, that is some seriously good target WiKi page. Master template for all others.
 
But I still do not see the circuit before me, uni-directional inverted S-bus and bi-directional Spektrum data line on the same UART Rx line.... I would get it if you where supposed to remove the SBUS jumper for Sat, but I read the opposite on RCG. Puzzling. A drawing might help understanding this.  But that might be a design secret.... See my confusion attached. 
![omnibus_sat_sbus](https://cloud.githubusercontent.com/assets/15121917/23063332/4c493c46-f50a-11e6-8104-e26619771181.jpg)